prettier.config.js

```diff
@@ -21,5 +21,6 @@ module.exports = {
                 proseWrap: "preserve"
             }
         }
-    ]
+    ],
+    filepath: "./packages/**/{src,test}/**/*.{js,jsx,ts,tsx,scss}"
```

Comment:
Is this just to include `scss` files or is that more to it?

Reply:
more 🤭

Reply:
What? 😄

Reply:
By moving this here `pretty-quick` respects the `filepath` and formats staged scss files, whereas previously it was just formatting (I think) js and ts.